react-components/src/hooks/useSearchMappedEquipmentFDM.tsx

```diff
@@ -446,6 +451,11 @@ async function filterSearchResultsByMappedTo3DModels(
     .map((node) => getDirectRelationProperties(node))
     .flat();
 
+  if (searchResultsNodes.length === 0) {
+    console.log('Search resutls are empty', searchResults);
```

Comment:
Do you want to keep this as a `console.log`? This sounds like something that should either be a `console.warn` or something that should be dropped it that is too spammy

Reply:
Yeah, just forgot to remove it. Removed it now